### ISSUE TITLE
How to get some host car data in global coordinates?

### ISSUE THREAD

**Issue by karlji1021** [yanpanlau/DDPG-Keras-Torcs#49]
I am wondering how to get get global coordinates for the host car, such as X, Y, and Z.
(I think z position can be got by using the sensor value)
I need to calculate acceleration and jerk values from global coordinates data,
so I want to pull the concerned values from TORCS environment.

In cpp files, "car->_accel_x" means host car's X-axis acceleration in global coordinates.
And, if possible, I hope to get "car->_yaw", "car->_yaw_rate", "car-> _accel_y".
However, I have trouble in getting these values from cpp file to python file.

Could anyone help me for this problem?

Thanks

**Comment by wuhuccp** [comment]
I also have the problem.Did you solve it?

**Comment by karlji1021** [comment]
> I also have the problem.Did you solve it?

No, I didn't solve it yet.. If you did it, could you let me know, please?
